tests/PhpSpreadsheetTests/Worksheet/WorksheetTest.php

```diff
@@ -181,4 +181,95 @@ public function testRemoveCellsCorrectlyWhenRemovingRow()
             $worksheet->getCell('C1')->getValue()
         );
     }
+
+    public function removeColumnProvider(): array
+    {
+        return [
+            'Remove first column' => [
```

Comment:
Nice use of index labelling for test data; I'd only recently discovered this feature of PHPUnit for labelling data from providers myself, and started to apply it in a few places, so it's good to see somebody else using it

Reply:
Yes, very handy indeed! Not only to quickly see which data set failed in case of any errors but also to know what's being tested. Looking at a random array of values makes it hard for your future self or contributors in this case.